src/plugins/checkboxes/checkbox.scss

```
@@ -0,0 +1,137 @@
+.provisional {
```

This could potentially affect other provisional features related to forms, like wb-postback. Since wb-postback is provisional the form will get those new styles and it could be unwanted.

---

I will scope it a little bit more:
```suggestion
.provisional {
   .chckbxrdio-grp label {
```
Would that make sense?

---

Still too broad